apps/web/.env.example

```
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_URL="http://localhost:3000"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)